test/tojs.js

```diff
@@ -0,0 +1,20 @@
+var test = require('tape');
```

Comment:
I would place this test right after this one: https://github.com/maxdeviant/mobx/blob/d0404bb0e4d95c5a5ea5f888ee5952ed4b256f6e/test/observables.js#L1074